.github/workflows/configure.sh

```diff
@@ -73,5 +73,5 @@ ${KRATOS_CMAKE_OPTIONS_FLAGS} \
 -DCMAKE_UNITY_BUILD=ON \
 -DINCLUDE_MMG=ON                                    \
 
-# Buid
+# Build
```

Comment:
Line 8 can be changed to:
You can find a list `with` all the `compilation` options in INSTALL.md or here: